pytorch_to_returnn/torch/nn/modules/operator.py

```diff
@@ -394,6 +406,24 @@ def create_returnn_layer_dict(self, input: Tensor) -> Dict[str, Any]:
     return {"class": "reduce", "mode": self.mode, "axes": self.axes, "from": self._get_input_layer_name(input)}
 
 
+class Length(Module):
+  """
+  Wraps RETURNN LengthLayer.
+  """
+  is_original_torch_module = False
+
+  def __init__(self, axis: Union[str, Dim, int]):
+    super(Length, self).__init__()
+    self.axis = axis
+
+  def create_returnn_layer_dict(self, input: Tensor) -> Dict[str, Any]:
+    axis = self.axis
+    if isinstance(axis, int):
+      axis = self._get_input_axis_to_returnn(input, axis)
+    return {
+      "class": "length", "axis": axis, "from": self._get_input_layer_name(input)}
```

Comment:
With `axis` always being an `int`, this becomes:
```suggestion
    return {
      "class": "length", "from": self._get_input_layer_name(input),
      "axis": self._get_input_axis_to_returnn(input, self.axis)}
```